MUXSDKKaltura/Podfile

```diff
@@ -4,13 +4,13 @@ target 'MUXSDKKaltura' do
   use_frameworks!
   
   # Pods for MUXSDKKaltura
-  pod 'Mux-Stats-Core', '~>3.5'
```

Comment:
Updated it actually needs 3.11 to support inferred key. It wouldn't crash or fail or anything like that but it wouldn't support it correctly.  Since this is the only feature added for this new release I increased the min version to ensure we support inferred key correctly.